doc-content/kogito-docs/src/main/asciidoc/drl/proc_drl-rules-create.adoc

```diff
@@ -196,16 +199,16 @@ import org.kie.kogito.rules.DataSource;
 import org.kie.kogito.rules.DataStream;
 
 declare MortgageRules extends RuleUnitData
-  bankruptcy: DataStream<Bankruptcy> = DataSource.createStream()
-  applicant: DataStream<Applicant> = DataSource.createStream()
-  application: DataStream<LoanApplication> = DataSource.createStream()
+  bankruptcy: DataStore<Bankruptcy> = DataSource.createStore()
+  applicant: DataStore<Applicant> = DataSource.createStore()
+  application: DataStore<LoanApplication> = DataSource.createStore()
 end
 
 rule "Bankruptcy history"
 	salience 10
 	when
-		$a : /loanApplication
-		exists (/bankruptcy[ yearOfOccurrence > 1990 || amountOwed > 10000 ])
+		$a : /loanApplication[ applicantName: applicant ]
+		exists (/bankruptcy[ name == applicantName, yearOfOccurrence > 1990 || amountOwed > 10000 ])
```

Comment:
I changed these rules and the corresponding declared types for the same reason as before: I added join conditions to evidence the relationships between facts.

Reply:
Done